### ISSUE TITLE
Filter Out Tracks Events for Tracks

### ISSUE THREAD

**Issue by that-mike-bal** [Automattic/tracks-chrome-extension#12]
This extension was super useful today and I wish I had started using it sooner. 

Idea for you: 

I noticed when I have many tabs open and I'm going back and forth between the page where I'm watching the event and a tab with Tracks open, I am getting Inceptionlike tracking for me using Tracks. 

It would be nice to filter those out or collapse them — idk if anyone else finds it useful unless they are actually working on tracking for Tracks but it was clouding up the events I was trying to find/pinpoint.

**Comment by AllTerrainDeveloper** [comment]
@that-mike-bal I am not quite following the issue, sorry! Would you mind elaborating or giving more hints using screenshots?

**Comment by that-mike-bal** [comment]
Yep so the scenario is that I'm using Vigilante for a specific tab to identify/verify events and then using Tracks in another tab to look at trends, props, etc. 

There are a number of extra events that fire when I open up tracks so I end up losing my place and the set of events I'm trying to focus on. 

Being able to filter or choose a tab for the stream would be great and help avoid too much build up.

**Comment by AllTerrainDeveloper** [comment]
Thank you! [I've started the development of version 2](https://leterop2.wordpress.com/2023/03/02/letero-weekly-update-feb-27-mar-3/#comment-768). This will be the main feature of it, a part of moving to React and TypeScript, so more powerful features can be added easier.
